src/query/builder.js

```diff
@@ -152,6 +154,20 @@ assign(Builder.prototype, {
     return this;
   },
 
+  // Adds a comment to the query
+  comment(txt) {
+    if (!isString(txt)) {
+      throw new Error('Comment must be a string');
+    }
+    if (txt.indexOf('/*') !== -1 || txt.indexOf('*/') !== -1) {
```

Comment:
you can use [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) whichi already returns a boolean and is a bit more declarative (although we could argue indexOf is known by everyone)

```suggestion
    if (txt.includes('/*') || txt.includes('*/')) {
```